cmd/agent/install_mac_os.sh

```diff
@@ -74,6 +74,20 @@ function find_latest_patch_version_for() {
     echo "$latest_patch"
 }
 
+function prepare_dmg_file() {
+    $sudo_cmd rm -f $dmg_file
```

Comment:
I'm not sure it's worth it, but you could at least have `$dmg_file` as a local variable in the method, wdyt?